skins/uncyclopedia/main.css

```diff
@@ -1771,7 +1771,7 @@ p.mw-ipb-conveniencelinks {
 
 #notifyForm input.submit
 {
-  background: url('http://images.wikia.com/common/extensions/wikia/SendToAFriend/SendToAFriendSubmitBg.png');
+  background: url('https://images.wikia.nocookie.net/common/extensions/wikia/SendToAFriend/SendToAFriendSubmitBg.png');
```

Comment:
We don't have this extension anymore, so I would remove this style.

Reply:
Good catch, thanks. And done. :)